app/components/visualizer/main-viz/templates/Chart.jsx

```diff
@@ -0,0 +1,42 @@
+import DynamicPieChart from "./DynamicPieChart.js";
+
+class Chart extends React.Component {
+  componentDidMount() {
+    let el = React.findDOMNode(this);
+
+    $.getJSON("blobs/deptGrades.json", (data) => {
+      this.chart = new DynamicPieChart({
+        id: "_id",
+        data: data,
+        element: "#main-viz",
+        height: 512,
+        width: 1024,
```

Comment:
I'll take a look at that article later, but I think we could easily add some resizing logic for when rendering happens.  JQuery can do this kind of stuff easily.